### DISCUSSION TITLE
新游以太战线正式登陆全平台！

### DISCUSSION THREAD

**Discussion by MRScongZZZ** [share121/inter-knot#1405] [其他]
![image](https://github.com/user-attachments/assets/0ed912ec-9523-41d3-bee7-63dcece261da)
成为首席训练师，与您的以太灵们夺得桂冠！

**Comment by XxnjiIAJ** [comment]
就决定是你了，眠眠！

**Comment by Gmadelyn7** [comment]
我要玩，我现在就要玩到以太战线

**Comment by lllliufeng** [comment]
来了来了

**Comment by gaojiaobei** [comment]
点击下方链接观看视频，教你获得双属性以太灵
链接：https://www.bilibili.com/video/BV1BH4y1c7

**Comment by lllliufeng** [comment]
楼上链接挂辣

**Comment by WORLD-executeME** [comment]
玩过内测，感觉...不如帝垣琼玉

**Comment by gaojiaobei** [comment]
感觉不如帝垣琼玉......游戏性

**Comment by kallen1314** [comment]
感觉不如折纸小鸟对对碰……画质

**Comment by Jiucheng258** [comment]
哼哼，首席训练师在此。

**Comment by xioagong** [comment]
不准你玩

**Comment by Dave-12138** [comment]
![image](https://github.com/user-attachments/assets/0e98ba18-2f40-423b-80bf-d31969dacd3a)
